doc/guides/contributing/pull-requests.md

```diff
@@ -356,10 +348,10 @@ your pull request waiting longer than you expect, see the
 
 When a collaborator lands your pull request, they will post
 a comment to the pull request page mentioning the commit(s) it
-landed as. GitHub often shows the pull request as `Closed` at this
+landed as. GitHub may show the pull request as `Closed` at this
```

Comment:
Nit: Microsoft style guide says to use _might_ rather than _may_ in this kind of situation.

```suggestion
landed as. GitHub might show the pull request as `Closed` at this
```

Ref: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may